10.0/apache/Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN a2enmod rewrite
 ENV NEXTCLOUD_VERSION 10.0.5
 VOLUME /var/www/html
 
+COPY config/* /usr/src/nextcloud/config/
```

Comment:
`COPY` has annoying and often inconsistent cache behavior -- any particular reason this needs to be moved up here and can't stay down below with the other `COPY`?